harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
ok - in that case, then `getOwnPropertyDescriptor` won't work in cases where the property is on an object higher up the prototype chain.

We'd have to walk up the [[Prototype]], and then preserve the object and descriptor that owned the property - so they can be restored later after the `delete`.